src/theme-preval.ts

```diff
@@ -348,11 +349,16 @@ const stateLabels = {
   }
 }
 
+const primerPrimitivesColors = splitCompoundColor(primitives.colors.light)
```

Comment:
What does this `splitCompoundColor` function do?

Reply:
@colebemis it recursively dives through an object and splits out all of the css values that have color and shadow parameters into separate keys - I thought we discussed something like this would be necessary?

So `shadow: 'inset 0 1px 2px rgba(27,31,35,0.075)'` becomes:

```
shadow: 'inset 0 1px 2px'
shadowColor: 'rgba(27,31,35,0.075)'
```

is this about what you had in mind when we discussed this the other day?

Reply:
Ah, I should have been more clear. They'll need to be entirely separate objects:

```js
// before
{
  colors: {
    btn: {
      text: '#24292e',
      shadow: '0 1px 0 rgba(27,31,35,0.04)'
    }
  }
}
```

```js
// after
{
  colors: {
    btn: {
      text: '#24292e',
    }
  },
  shadows: {
    btn: {
       shadow: '0 1px 0 rgba(27,31,35,0.04)'
    }
  }
}
```
```

Reply:
Also, I think we should exclude all of the `scale` colors from the `functionalColors` object. We want to discourage consumers from using those values directly.

Reply:
ahh, okay - that makes sense! I'll get working on pushing that up!